dbt/adapters/clickhouse/impl.py

```diff
@@ -484,17 +484,23 @@ def run_sql_for_tests(self, sql, fetch, conn):
         finally:
             conn.state = 'close'
 
+    def _build_settings_str(self, settings: Dict[str, Any]) -> str:
+        res = []
+        for key in settings:
+            if isinstance(settings[key], str) and not settings[key].startswith("'"):
+                res.append(f" {key}='{settings[key]}'")
+            else:
+                res.append(f" {key}={settings[key]}")
```

Comment:
Can we add an indication to the workaround support in the code? (will be way easier with the next person reading this code) something like this
```suggestion
                # support old workaround https://github.com/ClickHouse/dbt-clickhouse/issues/240#issuecomment-1894692117
                res.append(f" {key}={settings[key]}")
```